cmake/projects/libxml2/scripts/patched_bootstrap.bat.in

```diff
@@ -0,0 +1,4 @@
+call "@HUNTER_MSVC_VCVARSALL@" @HUNTER_MSVC_ARCH@
+cd @HUNTER_PACKAGE_SOURCE_DIR@/win32
```

Comment:
@xsacha See:
* https://github.com/ruslo/hunter/pull/1272/files#diff-818e35492b0123085e501510d3efee91

I think we need same patch here.